src/cljs/org/broadinstitute/firecloud_ui/page/workspace/submissions.cljs

```diff
@@ -0,0 +1,119 @@
+(ns org.broadinstitute.firecloud-ui.page.workspace.submissions
+  (:require
+    [dmohs.react :as react]
+    [org.broadinstitute.firecloud-ui.common.components :as comps]
+    [org.broadinstitute.firecloud-ui.common.style :as style]
+    [org.broadinstitute.firecloud-ui.paths :as paths]
+    [org.broadinstitute.firecloud-ui.utils :as utils]))
+
+(defn submission-running? [submission]
+  (let [status (get submission :status)]
+    (= status "Running")))
+
+(defn- create-mock-submissions-list [workspace]
+  (map
+    (fn [i]
+      {:workspaceName {:namespace (get workspace "namespace")
+                       :name (get workspace "name")}
+       :methodConfigurationNamespace "my_test_configs"
+       :submissionDate (str "2015-" (rand-nth (range 1 12)) "-"
```

Comment:
This is generating an entire range, then selecting one element from it.  To generate a random number from 1 to n (inclusive), just do `(inc (rand-int n))`.